src/View/Helper/FormHelper.php

```diff
@@ -24,6 +24,7 @@
 use Cake\View\Helper;
 use Cake\View\StringTemplateTrait;
 use Cake\View\View;
+use Cake\View\Widget\WidgetLocator;
 use Cake\View\Widget\WidgetRegistry;
```

Comment:
I think this line can be removed now.

Reply:
I leave it due a phpstan check, (a widgetLocator setter with typehint for BC)

Reply:
Ah sorry, it is needed for BC.